src/main/steps/common/common.content.ts

```diff
@@ -165,8 +166,9 @@ const cy: typeof en = {
     month: 'Mis',
     year: 'Blwyddyn',
   },
-  yes: 'Do',
```

Comment:
Was this already approved Welsh lang? we could just add the Not Sure option if so

Reply:
Yeah it was already approved welsh but text for some other application, hence I changed it to 'Yes (in welsh)',